packages/localizations/src/en-US.ts

```diff
@@ -487,6 +544,11 @@ export const enUS: LocalizationResource = {
     form_username_invalid_length: '',
     identification_deletion_failed: 'You cannot delete your last identification.',
     not_allowed_access: '',
+    organization_domain_blocked: '',
+    organization_domain_common: 'Esse é um domínio comum, por favor use um diferente.',
+    organization_membership_quota_exceeded: '',
+    organization_minimum_permissions_needed:
+      'É necessário que haja pelo menos um membro da organização com as permissões mínimas necessárias.',
```

Comment:
@LauraBeatris this should be in english, i believe this was added to create the images in the description of the PR

Reply:
Oh no, it was supposed to be empty strings but I forgot to update while testing it 😓